package.json

```diff
@@ -16,21 +16,24 @@
     "test": "cross-env NODE_CONFIG_DIR='./tests/e2e-tests/config' BABEL_ENV=commonjs mocha \"tests/e2e-tests\" --require babel-register --recursive",
```

Comment:
This and the two other below can be removed since it references the old e2e tests?

Reply:
@kloon you are right. I removed the legacy e2e test scripts.